### ISSUE TITLE
Implementation detail about LDAM loss

### ISSUE THREAD

**Issue by fliman** [Vanint/TADE-AgnosticLT#5]
Hi Vanint,
  I notice that in your LDAM loss implementation the scale is applied on the  the adjustment only
```
x_m = x - batch_m * self.s 
```
which is different from the original LDAM loss
```
 return F.cross_entropy(self.s*output, target, weight=self.weight)
```
basically equivalent to 
```
x_m = (x - batch_m) * self.s 
```
could you explained more on this detail? Is the coefficient absorbed somewhere on the logit output?

**Comment by YifanZhang07** [comment]
Hi, thanks for your attention. In fact, we did not use LDAM loss in our work. The provided LDAM loss is provided by the RIDE (https://github.com/frank-xwang/RIDE-LongTailRecognition/blob/main/model/loss.py). You may wanna leave a question in the repository of RIDE. Thanks.

**Comment by YifanZhang07** [comment]
If any other questions, happy to discuss them further.
